MetovaTestKitTests/Internal Utilities Tests/QuotedStringTests.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  QuotedStringTests.swift
+//  MetovaTestKit
+//
+//  Created by Logan Gauthier on 7/11/17.
+//  Copyright © 2017 Metova. All rights reserved.
+//
+//  MIT License
+//
+//  Permission is hereby granted, free of charge, to any person obtaining
+//  a copy of this software and associated documentation files (the
+//  "Software"), to deal in the Software without restriction, including
+//  without limitation the rights to use, copy, modify, merge, publish,
+//  distribute, sublicense, and/or sell copies of the Software, and to
+//  permit persons to whom the Software is furnished to do so, subject to
+//  the following conditions:
+//
+//  The above copyright notice and this permission notice shall be
+//  included in all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+//  EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+//  MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+//  NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+//  LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+//  OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+//  WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+//
+
+import XCTest
+
+@testable import MetovaTestKit
+
+class QuotedStringTests: MTKBaseTestCase {
+    
+    func testQuotedString() {
```

Comment:
Yea, I think so. Here's a question for you though. What would you expect the output of that to be? Should it simply add quotes around it again even if it already has quotes, or should it detect that it has quotes and not modify the original string? I'm leaning towards adding the extra quotes.